route_reflector/encode_originator_clist.c

```diff
@@ -12,8 +13,45 @@
 
 #define ORIGINATOR_ATTR_LEN 4
 
+PROOF_INSTS(
+
+        uint16_t nondet_len();
+        struct ubpf_peer_info *get_src_peer_info() {
+            struct ubpf_peer_info *pf;
+
+            pf = malloc(sizeof(*pf));
+            if (!pf) return NULL;
+
+            pf->local_bgp_session = malloc(sizeof(struct ubpf_peer_info));
+
+            pf->peer_type = IBGP_SESSION;
```

Comment:
It's better to put a non deterministic value instead of hardcoding the type of session